tst/Bootstrap.php

```diff
@@ -238,6 +238,17 @@ public static function getCommentJson()
         return json_encode(self::getCommentPost());
     }
 
+    /**
+     * Returns 16 random hexadecimal characters.
+     *
+     * @return string
+     */
+    public static function getRandomId()
+    {
+        // 8 binary bytes are 16 characters long in hex
+        return bin2hex(random_bytes(8));
```

Comment:
FYI ah compared to the old method, this also seems you've modernized the code. :+1:

Reply:
Discovered this code got duplicated, so better to use only one variant in multiple locations. Used the IMO more elegant version.